web/src/main/java/org/springframework/security/web/util/OnCommittedResponseWrapper.java

```diff
@@ -29,9 +29,12 @@
  *
  * @since 4.0.2
  * @author Rob Winch
+ * @author Eddú Meléndez
  */
 public abstract class OnCommittedResponseWrapper extends HttpServletResponseWrapper {
 
+	private static final String CR_OR_LF = "\r\n";
```

Comment:
Is there a reason you made the change here? I think the best place for the change is in `FirewalledResponse`. You would override both setHeader and addHeader methods of `HttpServletResponseWrapper`